opencog/nlp/scm/nlp-fc.scm

```diff
@@ -56,3 +111,11 @@
   parse-node
   )
 )
+(define (run-fc-dbg sent)
```

Comment:
Hi Amen,
 Rules are commented because the files are there but have issues in scm
code, so they don't work.
old files were kept in case we need to revert on something, but i will
remove those in future when I make the unit tests for nlp-fc code.
Regards,
Mandeep

On Thu, Aug 20, 2015 at 12:58 PM, Amen Belayneh <notifications@github.com>
wrote:

> In opencog/nlp/scm/nlp-fc.scm
> <https://github.com/opencog/opencog/pull/1734#discussion_r37495290>:
>
> > @@ -56,3 +111,11 @@
> >    parse-node
> >    )
> >  )
> > +(define (run-fc-dbg sent)
>
> please add comment on using this function
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/opencog/pull/1734/files#r37495290>.
>